README.md

```diff
@@ -143,6 +143,8 @@ By enabling JUnit 5 for instrumentation tests, you will gain access to `Activity
 
 ### Extensions
 
+#### GrantPermissionExtension
+
```

Comment:
I'm a bit torn on the addition of this subsection. It makes sense that we add this when there are more extensions below, and it's also true that `includeExtensions.set(true)` will only give you this particular extension right now. However, the original idea was to include other optional stuff, too. Therefore, I'd like to suggest a different name for it, such as "First-party extensions" or "Official extensions". I don't love either of these, so if you can come up with a better expression for "Extensions that are maintained within this repo, but are optional", please feel free and go ahead.

Reply:
You're right, my addition was placed in a wrong place. Caption "Extensions" grabbed my attention and I was too lazy to read carefully. 

I moved it down below, after the `Official support` chapter. I made it a section called "Support for Rules". This way you can later add more references, effectively making those extensions "officially approved by MannOderMaus" :) 

Later this chapter can be extracted to a sub-README if it grows to big.

Does this approach look better to you?
Feel free to comment/correct/rewrite/delete

Reply:
Force-pushed the reworked version.
Please review